pyrasterframes/src/main/python/docs/time-series.pymd

```diff
@@ -33,16 +33,17 @@ First we will get the vector data from LandsNet service by a REST query. The dat
 
 ```python get_park_boundary
 import requests
+nps_filepath = '/tmp/parks.geojson'
 nps_data_query_url = 'https://services1.arcgis.com/fBc8EJBxQRMcHlei/arcgis/rest/services/' \
                      'NPS_Park_Boundaries/FeatureServer/0/query' \
                      '?geometry=-82.451,41.075,-80.682,41.436&inSR=4326&outSR=4326&f=geojson'
 r = requests.get(nps_data_query_url)
```

Comment:
Did you try doing this with `SparkFiles` like this?

https://github.com/locationtech/rasterframes/blob/bdcfb39a4c8c7276e9a6c8d0ae41d81fd31f54df/pyrasterframes/src/main/python/docs/vector-data.pymd#L15-L18

Not requiring the use of `requests` (even though it is a nice library) would clear a bit more cognitive load.

Reply:
Just saw the comments in #288 which seem to indicate you tried the SparkFiles thing. I suspect the problem is the query parameter?

Reply:
The problem seems to be that SparkFiles does not know what the filename should be, but it is opaque actually. The call to "Add" the file seems to work or fail silently. But reading it doesn't and you can kind of peek at the local cache to see nothing is there.